docs/source/working_with_aiida/caching.rst

```diff
@@ -65,88 +113,69 @@ This means that caching is enabled for ``TemplatereplacerCalculation`` and ``Str
+    In [2]: from aiida.orm import Str
 
-Additionally, there are two methods you can use to disable caching for particular nodes:
+    In [3]: n1 = Str('test string')
 
-* The :meth:`~aiida.orm.nodes.Node.is_valid_cache` property determines whether a particular node can be used as a cache. This is used for example to disable caching from failed calculations.
-* Node classes have a ``_cachable`` attribute, which can be set to ``False`` to completely switch off caching for nodes of that class. This avoids performing queries for the hash altogether.
+    In [4]: n1.store()
+    Out[4]: u'test string'
 
-There are two ways in which the hash match can go wrong: False negatives, where two nodes should have the same hash but do not, or false positives, where two different nodes have the same hash. It is important to understand that false negatives are **highly preferrable**, because they only increase the runtime of your calculations, as if caching was disabled. False positives however can break the logic of your calculations. Be mindful of this when modifying the caching behaviour of your calculation and data classes.
+    In [5]: n2 = Str('test string')
 
-.. _caching_error:
+    In [6]: n2.store(use_cache=True)
+    Out[6]: u'test string'
 
-What to do when caching is used when it shouldn't
--------------------------------------------------
+    In [7]: print('UUID of n1:', n1.uuid)
+    UUID of n1: 956109e1-4382-4240-a711-2a4f3b522122
 
-In general, the caching mechanism should trigger only when the output of a calculation will be exactly the same as if it is run again. However, there might be some edge cases where this is not true.
+    In [8]: print('n2 is cached from:', n2.get_extra('_aiida_cached_from'))
+    n2 is cached from: 956109e1-4382-4240-a711-2a4f3b522122
 
-For example, if the parser is in a different python module than the calculation, the version number used in the hash will not change when the parser is updated. While the "correct" solution to this problem is to increase the version number of a calculation when the behavior of its parser changes, there might still be cases (e.g. during development) when you manually want to stop a particular node from being cached.
+When running a ``CalcJob`` through the ``Process`` interface, you can achive the same effect by passing ``_use_cache=True`` to the ``run`` or ``submit`` method.
 
-In such cases, you can follow these steps to disable caching:
+If you suspect a node is being reused in error (e.g. during development),
+it is also possible to manually *prevent* a specific node from being reused:
 
-1. If you suspect that a node has been cached in error, check that it has a ``_aiida_cached_from`` extra. If that's not the case, it is not a problem of caching.
-2. Get all nodes which match your node, and clear their hash:
+1. Check that the node in question has a ``_aiida_cached_from`` extra. 
+   If that's not the case, it anyhow is not reused.
```

Comment:
"that is" instead of "that's". Contractions are chat language or flavour for novel writers, if you like :smile:

Reply:
Also the second bi-sentence should be reworded: "it is anyhow not reused".